File: Problems-on-string-in-cpp/problem42.cpp
/////////////////////////////////////////////////////////////////////////////////////////////////////////////
// Write a program which accept string from user and check whether it contains vowels in it or not.
//
// Input:
//
// "marvellous"
//
// Output:
//
// TRUE
//
// Input:
//
// "Demo"
//
// Output:
//
// TRUE
//
// Input:
//
// "xyz"
//
// Output:
//
// FALSE
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include <iostream>
using namespace std;

bool CountCapital(char *str)
{
    bool BRET = false;

    while (*str != '\0')
    {
        if (*str == 'a' || *str == 'e' || *str == 'i' || *str == 'o' || *str == 'u' || *str == 'A' || *str == 'E' || *str == 'I' || *str == 'O' || *str == 'U')
        {
            BRET = true;
        }
        else
        {
            BRET;
        }
        str++;
    }

    return BRET;
}

int main()
{
    char Arr[20];
    bool bRet = 0;

    cout << "Enter string\n";
    cin.getline(Arr, 20);

    bRet = CountCapital(Arr);
    if (bRet == true)
    {
        cout << "String contains vovels in it\n";
    }
    else
    {
        cout << "String does not contain vovels in it\n";
    }

    return 0;
}

File: Problems-on-Bitwise/problem42.java
////////////////////////////////////////////////////////////////////////////////
// 
// Write a program which accept one number from user and check 
// whether 9th and 12th bit is on or off.......
// 
//  Input -257
//  Output -True
///////////////////////////////////////////////////////////////////////////////


import java.lang.*;
import java.util.*;

class Bitwise
{
    public boolean ChkBit(int iNo)
    {
        int iMask=0X00000900;
        int iResult=0;

        iResult=iNo&iMask;
        if(iResult==0)
        {
            return false;
        }
        else
        {
            return true;
        }

    }
}



class program3_33
{
    public static void main(String args[])
    {
        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter number");
        int iValue=sobj.nextInt();


        Bitwise bobj=new Bitwise();
        boolean bRet=bobj.ChkBit(iValue);

        if(bRet==true)
        {
            System.out.println("Bit is on");
        }
        else
        {
            System.out.println("Bit is off");
        }


    }
}